lesson-02-assignment/lesson-02-assignment.rb

```diff
@@ -1,16 +1,19 @@
 #1
 
-Course.create(coding_class_id: "1", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "2", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "3", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "4", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "5", trimester_id: "8", max_enrollment: "25")
+(1..5).each do |id|
+Course.create(
+coding_class_id: id,
+trimester_id: 8,
+max_enrollment: 25
+)
+end
 
 #2
 
 Student.create(first_name: "Val", last_name: "Lyzhyn", email: "val@test.com")
 student_id = Student.last.id
-Enrollment.create(course_id: "1", student_id: student_id)
+course = trimester = Trimester.find_by(year: 2026, term: "Spring")
+Enrollment.create(course_id: course.id, student_id: student_id)

```

Comment:
In this case, `course` is effectively a Trimester, which isn't what you want.  Here you want to retrieve all the Courses that have the id of the Trimester that is in Spring 2026, so you do want `trimester = Trimester.find_by(year: 2026, term: "Spring")`.

You can then use `trimester` to look up the courses you want with something like `courses = Course.where(trimester_id:....)`.  

Then you can loop through `courses` and create an enrollment for each one, similar to my suggestion above.